listenbrainz/webserver/views/user_settings_api.py

```diff
@@ -27,6 +27,8 @@ def update_flair():
     if "flair" not in request.json:
         raise APIBadRequest("Missing flair")
 
+    # todo: add flair validation
```

Comment:
Does this need to be done now or add a ticket?

Reply:
this needs to be done (if at all) after the structure of the flairs has been decided.

Reply:
OK, sounds like no validation will be needed, so lets remove the comment on move on.